apostrophecms/apostrophe
# add close on image control dialog in rich text

## Summary

Allow to close the image modal in rich text widgets

## What are the specific steps to test this change?

1. Add `image` in the `toolbar` of a rich-text widget configuration
2. Select some text in the widget and click on the image icon
3. You should be able to close the extra modal by
   1. clicking on the `Cancel` button
   2. clicking anywhere on the page
   3. clicking somewhere else in the rich-text area
   4. selecting an image and clicking `Save`
4. You compare this with the link feature if you want

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
